src/Tests/Microsoft.NET.TestFramework/ExponentialRetry.cs

```diff
@@ -19,13 +19,13 @@ public static IEnumerable<TimeSpan> Intervals
                 while (true)
                 {
                     yield return TimeSpan.FromSeconds(seconds);
-                    seconds *= 2;
+                    seconds *= 10;
```

Comment:
Looks like this will wait 5 seconds, 50 seconds, 500 seconds, 5000 seconds, then 50000 seconds.  50000 seconds is almost 14 hours, so some higher level timeout is almost certain to abort the test before then.  Would a factor of something like 5 be better?  Or maybe we don't want to do a full exponential backoff.  Maybe once we get to some retry delay (maybe 5 minutes?) we stop increasing the delay, or decrease the exponential factor to 1.5 or 2.

EDIT: Looks like you changed the maxRetryCount to 2.  I think we might want to add more retries but tweak the backoff strategy.  We might want to have a random element to the delay so that if a bunch of tests fail at once, they don't all retry at the exact same time.

Reply:
I don't think we need to spend too much effort on it. Given our failure rate is about 1%, the first retry will work. And our tests does not run at the same time